notes.md

```diff
@@ -51,6 +51,10 @@
 * The total number of shares outstanding is the difference between a cap and `token.totalSupply()` and is rightly a function of the `CrowdsaleContract`.
 * The date the share was purchased is a property of the block the purchase was processed in so there is no need to store it in the token. However the user database might wish to record that information as well for convenience.
 * The price the user paid is a function of the ETH price per Token (which can change during the crowdsale) and ETH price at that point (which can also be set in the `CrowdsaleContract`) and the number of ETH spent, which the `CrowdsaleContract` also knows and which is returned in the transaction receipt.
+* Tokens can only be minted in whole numbers so `floor` the conversion and put the remainder, less some gas, into a `RefundVault`.
+* Move the ETH to be turned into Tokens into the Crowdsale's `wallet`.
+* Move the remainder into a `RefundVault` called `refundWallet`.
+* People with refunds can claim their refunds through the Crowdsale Website.
```

Comment:
Are refunds typically something that are given automatically during the crowdsale or after the crowdsale is complete?

Reply:
After discussion with the lawyer we'll release funds at the end of the crowdsale